src/user_position.jl

```diff
@@ -54,7 +58,7 @@ function calc_DOP(H_GEO)
     HDOP = sqrt(D[1,1] + D[2,2]) # horizontal dop
     PDOP = sqrt(D[1,1] + D[2,2] + D[3, 3]) # position dop
     GDOP = sqrt(sum(diag(D))) # geometrical dop
-    return GDOP
+    return GDOP, PDOP, VDOP, HDOP, TDOP
```

Comment:
I`m note quite sure, but maybe we should split this into a function for each DOP or return a structure, what is your opinion on this @zsoerenm

Reply:
Yes, for the sake a clarity an extra struct would be helpful. At the moment all DOPs are saved in the PVTSolution individually. Those could be saved within a single struct, that is then part of PVTSolution with some helper functions to get the DOPs.